mingw-w64-git/PKGBUILD

```
@@ -0,0 +1,141 @@
+# Maintainer: Yuyi Wang <Strawberry_Str@hotmail.com>
+
+_realname=git
+pkgbase="mingw-w64-${_realname}"
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=2.39.2.windows.1
+pkgrel=1
+pkgdesc="The fast distributed version control system (mingw-w64)"
+arch=('any')
+url="https://git-for-windows.github.io/"
+license=('GPL2')
+groups=('VCS')
+provides=('git' 'git-core')
+
+options=('strip')
+
+makedepends=("diffutils"
+             "docbook-xsl"
+             "make"
+             "patch"
+             "xmlto"
```

```suggestion
makedepends=("${MINGW_PACKAGE_PREFIX}-docbook-xsl"
             "xmlto"
```
other packages are installed by default when installing base-devel.

---

OK, but I need the MSYS2 `docbook-xsl` to build docs.